Modix/ModixBot.cs

```diff
@@ -98,6 +99,9 @@ public async Task StartWebserver()
         {
             await _client.SetGameAsync("https://mod.gg/");
 
+            //TODO: Maybe implement this differently
+            _hooks.CurrentBotId = _client.CurrentUser.Id;
```

Comment:
The entire hooks system needs to be implemented differently. If this works for now, don't worry about it.